server/models/auth/signup.js

```diff
@@ -65,7 +65,7 @@ const emailMessage = (fullName, otp, port, host, username) => {
   return `<p>Hello ${fullName} !</p>
           <p>The OTP for verifying your email is ${otp}</p>
           <p>Please verify your email by visiting the following link</p>
-          <a href='http://${host}:${port}/auth/verify_email?username=${username}'>
+          <a href='http://${host}:${port}/auth/verify-email?username=${username}'>
```

Comment:
same

Reply:
Similar as above. The form for verification is available at http://localhost:3000/auth/verify-email?username=${username}

Reply:
The request from that form would be sent to the desired endpoint in server. There is no change in that.

Reply:
cool